src/coreclr/jit/lower.h

```diff
@@ -375,6 +375,18 @@ class Lowering final : public Phase
     void LowerModPow2(GenTree* node);
     GenTree* LowerAddForPossibleContainment(GenTreeOp* node);
 #endif // !TARGET_XARCH && !TARGET_ARM64
+
+    GenTree* InsertNewSimdCreateScalarNode(var_types type, GenTree* op1, CorInfoType simdBaseJitType, unsigned simdSize)
+    {
+        GenTree* result = comp->gtNewSimdCreateScalarUnsafeNode(TYP_SIMD16, op1, simdBaseJitType, 16);
```

Comment:
> Is it intentional both type and simdSize are always ignored here (and seemingly hardcoded to be TYP_SIMD16 / 16).

no, copy/paste error.

> Additionally, does simdSize need to be passed explicitly or it can be inferred as genTypeSize(type)?

In this case it can likely be implicitly inferred. But there are cases where that isn't true so we've consistently taken it explicitly.

Reply:
Fixed.